.github/workflows/deploy-live-cloud.yml

```
@@ -0,0 +1,24 @@
+      - name: Install sshpass
+        run: sudo apt-get update && sudo apt-get install -y sshpass
+
+      - name: SSH into VPS and deploy
+        env:
+          SSH_PASSWORD: ${{ secrets.VPS_PASSWORD }}
+        run: |
+          sshpass -p "$SSH_PASSWORD" ssh -o StrictHostKeyChecking=no root@${{ secrets.VPS_HOST }} << 'EOF'
+            cd /root/Rewatch/Kai-Voice-Call/
+            git pull origin main
+            lk agent deploy
+          EOF
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)